src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Data/SqlLite/00000000000000_CreateIdentitySchema.Designer.cs

```diff
@@ -1,20 +1,65 @@
-﻿using System;
+﻿// <auto-generated />
+using System;
+using Company.WebApplication1.Data;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
 using Microsoft.EntityFrameworkCore.Migrations;
 
 namespace Company.WebApplication1.Data.Migrations
 {
     [DbContext(typeof(ApplicationDbContext))]
-    [Migration("00000000000000_CreateIdentitySchema")]
+    [Migration("20180419233801_CreateIdentitySchema")]
```

Comment:
Is there anyway to hardcode this to 00000000000000_CreateIdentitySchema?

Reply:
Not from command-line. You can override the `IMigrationsIdGenerator` service...